docs/references/javascript/sign-in/sign-in.mdx

```diff
@@ -35,9 +35,15 @@ The `SignIn` object's properties can be split into logical groups, with each gro
 
 ### `create()`
 
-Use this method to kick-off the sign in flow. It creates a `SignIn` object and stores the sign-in lifecycle state.
+Use this method to initiate the sign-in process. It creates a new `SignIn` object and stores the sign-in lifecycle state.
 
-Depending on the use-case and the `params` you pass to the `create` method, it can either complete the sign-in process in one go, or simply collect part of the necessary data for completing authentication at a later stage.
+What you pass to `params` depends on the [configuration](/docs/authentication/configuration/sign-up-sign-in-options) of your application instance.
+
+The `create()` method will return a promise of the new `SignIn` object. This sign-in might be complete if you supply the required fields in one step. However, this is not mandatory. Clerk's sign-up process provides great flexibility and allows users to easily create multi-step sign-in flows.
```

Comment:
```suggestion
The `create()` method will return a promise of the new `SignIn` object. Optionally, you can complete the sign-in, fully authenticating the user, if you supply the required fields in one step. Otherwise, Clerk's sign-in process provides great flexibility and allows users to easily create multi-step sign-in flows.
```

Simplified some language and elaborated on what "complete" means in this context. Also, the original text here mentions "sign-up" process, but I corrected it to "sign-in". Not sure if that's right or wrong, just wanted to flag it as one of the changes here.